[quant][graphmode][fix] insert_prepack_unpack for conv

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27403 [quant][graphmode] Minor fix and turn off fold_convbn
* #27348 [quant][graphmode] Add e2e test for conv+bn
* #27347 [quant][graphmode][fix] Don't permute in quantized::conv2d pattern
* **#27346 [quant][graphmode][fix] insert_prepack_unpack for conv**

Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182915](https://our.internmc.facebook.com/intern/diff/D18182915)

jerryzh168 committed:
[quant][graphmode][fix] insert_prepack_unpack for conv

Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
How did we miss it in the first place?

jerryzh168 commented:
the test just checks the name of the swapped function, it didn't check the arguments

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182915](https://our.internmc.facebook.com/intern/diff/D18182915)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182915](https://our.internmc.facebook.com/intern/diff/D18182915)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182915](https://our.internmc.facebook.com/intern/diff/D18182915)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fix] insert_prepack_unpack for conv"


Summary:
att

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18182915](https://our.internmc.facebook.com/intern/diff/D18182915)

[ghstack-poisoned]
